website/www/site/content/en/documentation/sdks/python-machine-learning.md

```diff
@@ -165,7 +165,29 @@ For detailed instructions explaining how to build and run a pipeline that uses M
 
 ## Beam Java SDK support
 
-RunInference API is available to Beam Java SDK 2.41.0 and later through Apache Beam's [Multi-language Pipelines framework](https://beam.apache.org/documentation/programming-guide/#multi-language-pipelines). Please see [here](https://github.com/apache/beam/blob/master/sdks/java/extensions/python/src/main/java/org/apache/beam/sdk/extensions/python/transforms/RunInference.java) for the Java wrapper transform to use and please see [here](https://github.com/apache/beam/blob/master/sdks/java/extensions/python/src/test/java/org/apache/beam/sdk/extensions/python/transforms/RunInferenceTransformTest.java) for some example pipelines.
+The RunInference API is available with the Beam Java SDK versions 2.41.0 and later through Apache Beam's [Multi-language Pipelines framework](https://beam.apache.org/documentation/programming-guide/#multi-language-pipelines). For information about the Java wrapper transform, see [RunInference.java](https://github.com/apache/beam/blob/master/sdks/java/extensions/python/src/main/java/org/apache/beam/sdk/extensions/python/transforms/RunInference.java). For example pipelines, see [RunInferenceTransformTest.java](https://github.com/apache/beam/blob/master/sdks/java/extensions/python/src/test/java/org/apache/beam/sdk/extensions/python/transforms/RunInferenceTransformTest.java).
+
+## TensorFlow support
```

Comment:
@ryanthompson591 could you provide a canonical example here that includes the imports needed?

Reply:
I'm not sure what exactly you want as a canonical example. Do you mean sample code that would go into this doc, or something else?

Reply:
Yes something very simple that shows what imports are needed etc... It does not need show how to build a model but just enough to help folks not have to dig around to find the dependencies etc.. As this handler is a little different from the norm

Reply:
I added a section with TFx imports (below). Please review to make sure it's accurate.

Reply:
Note we can add any extra information in a separate PR.

Reply:
I took a look, agree with Reza that a simple example would be helpful here. 

How about this:

To use TensorFlow with the RunInference API, you need to:
- use `tfx_bsl==1.10.0` or newer
- create a model handler using `tfx_bsl.public.beam.run_inference.CreateModelHandler()`, 
- use the model handler with apache_beam.ml.inference.base. RunInference() transform.

A sample pipeline might look like the following:

```
import apache_beam as beam
from apache_beam.ml.inference.base import RunInference
from tensorflow_serving.apis import prediction_log_pb2
from tfx_bsl.public.proto import model_spec_pb2
from tfx_bsl.public.tfxio import TFExampleRecord
from tfx_bsl.public.beam.run_inference import CreateModelHandler

pipeline = beam.Pipeline()
tfexample_beam_record = TFExampleRecord(file_pattern=predict_values_five_times_table)
saved_model_spec = model_spec_pb2.SavedModelSpec(model_path=save_model_dir_multiply)
inference_spec_type = model_spec_pb2.InferenceSpecType(saved_model_spec=saved_model_spec)
model_handler = CreateModelHandler(inference_spec_type)
with pipeline as p:
    _ = (p | tfexample_beam_record.RawRecordBeamSource() 
           | RunInference(model_handler)
           | beam.Map(print)
        )
```

@ryanthompson591 can you test that this works?

Also there is a way how we configure unit tests for snippets to make sure they continue to work. Ideally we'd use that as well.

Reply:
Excellent suggestion. I made these changes.

Reply:
> @ryanthompson591 can you test that this works?

Did we do this?

Reply:
> file_pattern=predict_values_five_times_table)

predict_values_five_times_table, save_model_dir_multiply  is not defied in this snippet, so it's somewhat confusing.

Reply:
I agree that it's confusing; I think we're planning to add the notebook as soon as we have it ready.